cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImplConsistency.qll

```diff
@@ -0,0 +1,136 @@
+/**
+ * Provides consistency queries for checking invariants in the language-specific
+ * data-flow classes and predicates.
+ */
+
+private import DataFlowImplSpecific::Private
+private import DataFlowImplSpecific::Public
+private import TaintTrackingUtil
```

Comment:
This file does not exist in C#; it is `TaintTrackingPrivate.qll`/`TaintTrackingPublic.qll`.

Reply:
Perhaps we should agree on a `TaintTrackingImplSpecific.qll` file with modules `Public` and `Private`, like we have `DataFlowImplSpecific.qll`.

Reply:
We already have that in the `tainttracking*` subdirs, so I'll just use one of those.